src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -2106,6 +2120,8 @@ class Config : Vigilant(File("./config/skytils/config.toml"), "Skytils", sorting
 
         addDependency("showEtherwarpTeleportPosColor", "showEtherwarpTeleportPos")
 
+        addDependency("autoCopyVeryRareDrops", "autoCopyRNGDrops")
```

Comment:
Perhaps we could show both settings in case someone wants very rare but not crazy rare for some reason?

Reply:
> Perhaps we could show both settings in case someone wants very rare but not crazy rare for some reason?

_for some reason_

but Ok